### DISCUSSION TITLE
Metrics Endpoint

### DISCUSSION THREAD

**Discussion by codyshoffner** [requarks/wiki#7376] [Help / Questions] [answered]
I'd like to request having metrics endpoints exposed in the Wiki.Js Helm chart. It would be useful to send metrics to Prometheus. 

I have found this old discussion, but it did not gather any traction: https://github.com/requarks/wiki/discussions/6439

**Comment by NGPixel** [comment, marked as answer]
It has been added already in the upcoming 3.0 release.
